crates/hyperdrive-math/src/short/open.rs

```diff
@@ -98,7 +101,7 @@ impl State {
 mod tests {
     use std::panic;
 
-    use ethers::types::U256;
+    use ethers::types::{I256, U256};
```

Comment:
I don't see this import being used, was this residual?

Reply:
It's being used in the test below and was moved from the file's imports at the top, to the test's imports.